### PR TITLE
Unit Tests; 1 fix to appFooter debug menu

### PR DESCRIPTION
**Pull request by RaymondGrumney**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Your_Local_Install_to_Modify_Habitica%27s_Website_and_API for more info)

[//]: # (Put Issue # here, if applicable. This will automatically close the issue if your PR is merged in)

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)
Unit Tests

[//]: # (Put User ID in here - found on the Habitica website at User Icon > Settings > API)

----
UUID:

### CODE REVIEW

File: test/client/unit/specs/store/getters/user/userGems.js [hunk 1]
```diff
@@ -1,13 +0,0 @@
-import { gems as userGems } from 'client/store/getters/user';
```
Comment (paglias) [inline test/client/unit/specs/store/getters/user/userGems.js:1]: why was this removed?

**Review by paglias — CHANGES_REQUESTED** [review]
thanks! can you remove the changes to package-lock.json?

File: test/client/unit/specs/store/getters/user/userGems.js (hunk 1 above)
Comment (RaymondGrumney) [reply test/client/unit/specs/store/getters/user/userGems.js:1]: this test was folded into the user.test.js file. I probably should have just renamed the file.

File: test/client/unit/specs/components/avatar.vue.test.js [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+                  twoHanded: false,
+                },
+              },
+            },
+          },
+          preferences: { costume: false },
+        };
+      });
+
+      xit('returns false if not two handed weapon', () => {
```
Comment (paglias) [inline test/client/unit/specs/components/avatar.vue.test.js]: can you add a comment on why these are disabled? or if they can't be fixed, can you remove them? Thanks

**Review by paglias — CHANGES_REQUESTED** [review]

**Review by paglias — APPROVED** [review]

**Comment by paglias** [comment]
thanks @RaymondGrumney and sorry for the delay! If you have an Habitica UUID I'll be happy to note this contribution in your profile!
